README.md

```diff
@@ -1,2 +1,40 @@
-# quiz-mini-project
- A quizz app
+
+# Quiz App Screens
+
+These screens are mock screens intended to use in the Quiz React Application, made of plain HTML and CSS
+
+
+
+## P0 Screens
+1.Home Page\
+2.Quiz by category\
+3.Rules Page\
+4.MCQ based Quiz Questions\
+5.Results Page
+## P1 and P2 Screens
+1.P1 - Timer For Quiz\
+2.P2 - Create Quiz
```

Comment:
Mohammed just a small suggestion, it would be better if you could give links for this list item and you could also try writing items this way:-
- Home
- Quiz
- Rules

They would appear in bullet points if you put - infront of every list item and if you give them link as well then they will appear in blue color. I think then your readme would look much better.

Reply:
i'm planning to link those before the final deployment